adam/curriculum/verbs_with_dynamic_prepositions_curriculum.py

```diff
@@ -945,12 +956,14 @@ def _x_throws_y_to_z_template(
                 argument_roles_to_fillers=[
                     (AGENT, agent),
                     (THEME, theme),
-                    (GOAL_MANIPULATOR, recipient),
+                    # hack since English language generator doesn't generate correct language for GOAL_MANIPULATOR currently
+                    (GOAL, Region(recipient, distance=PROXIMAL)),
                 ],
             )
         ],
+        after_action_relations=[near(theme, recipient)],
         constraining_relations=flatten_relations(bigger_than([agent, recipient], theme)),
-        gazed_objects=[theme],
+        gazed_objects=[agent, theme, recipient],
```

Comment:
I realize this param is called `gazed_objects` but in the normal case we only want to be gazing at one object. We're set up to handle more than one to see if input noise on the `gaze` attribute causes learning issues.